linchpin/examples/example5/README.md

```diff
@@ -0,0 +1,178 @@
+
+```
+./kstests-in-cloud.sh provision nightly-permanent --pinfile examples/example5/PinFile --key-name kstests --key-use-existing --ansible-private-key ~/.ssh/kstests.pem --key-use-for-master
+```
+
+2) Add ssh key to *results host*
+
+The `kstests` public key needs to be added to *result host* so that *master* can rsync the results.
+
+3) Create test and scheduling configuration
+
+Copy the example configuration files to the ansible role's [location](../../../ansible/roles/kstest-master/vars/main) from which the role's defaults will be overriden.
+
+
+```
+cp linchpin/examples/example5/test-configuration.yml ansible/roles/kstest-master/vars/main/test-configuration.yml
+cp linchpin/examples/example5/schedule.yml ansible/roles/kstest-master/vars/main/schedule.yml
+```
+
+You may want to track the files in git repository (also to be aware that the default configuration will be actually overriden when running tests from this repository):
+```
+git add ansible/roles/kstest-master/vars/main/test-configuration.yml
+git add ansible/roles/kstest-master/vars/main/schedule.yml
+```
+
+4) Use the inventory generated with provisioning
+
+
+When applying the configuration for the first time it can be useful to set up the inventory for the local `kicstart-tests` repository so that it does not have to be passed to `ansible-playbook` with `-i` option each time the configuration is applied (or other playbook is run).
+
+To get the path of the `nightly-parmanent` *target*'s inventory run:
+
+```
+./kstests-in-cloud.sh status nightly-permanent
+```
+showing:
+```
+Target "nightly-permanent" has generated inventory linchpin/inventories/nightly-permanent.inventory
+```
+To configure ansible locally to use the inventory create local ansible configuration for the current directory:
+
+```
+echo -e "[defaults]\ninventory=linchpin/inventories/nightly-permanent.inventory" >  ansible.cfg
+```
+and perhaps add it to git for tracking
+```
+git add ansible.cfg
+```
+
+5) Apply the configuration to *master*:
+
+```
+ansible-playbook ansible/kstest-master-configure-test.yml
+```
+
+Or, if not using local `ansible.cfg` from 4., configure the inventory with `-i` option:
+```
+ansible-playbook -i linchpin/inventories/nightly-permanent.inventory ansible/kstest-master-configure-test.yml
+```
+
+6) Configure the scheduling on *master*:
+
+```
+ansible-playbook ansible/kstest-master-schedule-test.yml
+```
+Check that the scheduling was applied to *master*:
+```
+$ ansible kstest-master -a "crontab -l -u kstest" --become
+10.8.243.115 | CHANGED | rc=0 >>
+PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
+#Ansible: Scheduled kickstart tests
+5 0 * * * /home/kstest/run_tests.sh
+```
+30) When needed, update and apply the configuration again
```

Comment:
I don't understand this structure, why you have `1)` `2)`... `6)` and `30)` here. Is it on purpose? Maybe I just don't see that without formatting?

Reply:
Still waiting answer.

Reply:
That should suggest that some number of other steps may happen before 30)